pkgs/tools/typesetting/lowdown/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "lowdown";
-  version = "0.7.5";
+  version = "0.7.9";
 
   outputs = [ "out" "dev" ];
 
   src = fetchurl {
     url = "https://kristaps.bsd.lv/lowdown/snapshots/lowdown-${version}.tar.gz";
-    sha512 = "1wfbrydbk0f0blhg5my3m5gw8bspwh3rdg4w4mcalnrwpypzd4zrggc4aj3zm72c5jikx6pnjb2k9w1s075k84f6q8p8chlzb3s4qd2";
+    sha512 = "18q8i8lh8w127vzw697n0bzv4mchhna1p4s672hjvy39l3ls8rlj5nwq5npr5fry06yil62sjmq4652vw29r8l49wwk5j82a8l2nr7c";
```

Comment:
Why does this package use `sha512` instead of `sha256`?

Reply:
Because upstream offers sha512 sums which makes updating the hash very convenient, i. e. `nix-hash --type sha512 --to-base32 $(curl https://kristaps.bsd.lv/lowdown/snapshots/lowdown-${version}.tar.gz.sha512 | cut -d' ' -f2)`